spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -175,40 +173,99 @@ An example of an active channel on the controller chain can look like this:
 
-Due to how the mechanics of ICS-04 channel version negotiation operate the version passed into the host chain side (`OnChanOpenTry`) must match the counterparty version passed into the controller side (`OnChanOpenAck`) otherwise, there will be a resulting error at the IBC protocol level. 
+ICS-04 allows for each channel version negotiation to be application-specific. In the case of interchain accounts, the channel version will be a string of a JSON struct containing all the relevant metadata intended to be relayed to the counterparty during the channel handshake step ([see summary below](#Metadata-negotiation-summary)).
 
-Combined with the one channel per interchain account approach, this method of version negotiation allows us to pass the address of the interchain account back to the controller chain and create a mapping from controller portID -> interchain account address during the `OnChanOpenAck` callback. As outlined in the [controlling flow](#Controlling-flow), a controller chain will need to know the address of a registered interchain account in order to send transactions to the account on the host chain.
+Combined with the one channel per interchain account approach, this method of metadata negotiation allows us to pass the address of the interchain account back to the controller chain and create a mapping from controller portID -> interchain account address during the `OnChanOpenAck` callback. As outlined in the [controlling flow](#Controlling-flow), a controller chain will need to know the address of a registered interchain account in order to send transactions to the account on the host chain.
 
-#### Version negotiation summary
+### Metadata negotiation summary
 
 `interchain-account-address` is the address of the interchain account registered on the host chain by the controller chain.
 
-| Initiator | Datagram         | Chain acted upon |  Version (Controller, Host) |
-| --------- | ---------------- | ---------------- |  ---------------------- | 
-| Controller| ChanOpenInit     | Controller       |  (ics27-1, none)        | 
-| Relayer   | ChanOpenTry      | Host             |  (ics27-1, ics27-1.{interchain-account-address})        | 
-| Relayer   | ChanOpenAck      | Controller       |  (ics27-1, ics27-1.{interchain-account-address})        | 
-| Relayer   | ChanOpenConfirm  | Host             |  (ics27-1, ics27-1.{interchain-account-address})        | 
+**INIT**
+
+Initiator: Controller
+
+Datagram: ChanOpenInit
+
+Chain Acted Upon: Controller
+
+Version: 
+```json
+{
+  "Version": "ics27-1",
+  "Encoding": ["supported encoding types", "..."],
+  "Tx_Types": ["supported transaction types", "..."],
```

Comment:
I've included these two to be very explicit about any assumptions shared between the two chains. However, open to feedback on if we should standardize the encoding to all implementations.

Or if we should allow the tx_type to be implicit.

Note in the implementation there will only be one supported type for each

```
{
...
Encoding: protobuf,
Tx_Types: sdk_multi_msg,
}
```